pkg/util/util.go

```diff
@@ -164,3 +166,32 @@ func CreateTempDir(path string) (string, error) {
 	dir, pattern := getTempPathArgs(path)
 	return os.MkdirTemp(dir, pattern)
 }
+
+func IsOSTree() bool {
+	exists, err := PathExists("/run/ostree-booted")
```

Comment:
We have couple places in `pkg/admin/{prerun,autorecovery}/` that do the same thing, except they honor the `err`. Should we unify this a bit more?